README.md

```diff
@@ -24,33 +23,82 @@ This script should work for other types of stacks, but it hasn't been tested.
+
+These instructions should help you deploy paired sets of LARA & Portal servers.
+The examples assume the QA environment, but you can do this in staging as well.
+Recent template changes should make this process easier.
 
+Summary:
+  - Clone RDS instances of the production LARA and Portal databases using snapshots if you can.
+  - Use `create-stack` scripts to provision a Portal using configs and param overrides.
+  - Modify the Portal auth-client `authoring` configuration to fix authentication.
+  - Update URLS to shared resources on both servers using `lib/script/rewrite_lara_portal_resources.rb` from the LARA repo.
+
+### Creating the stacks
 1. Change into the `scripts` directory and install the dependencies: `cd scripts && npm install`
 2. Create some configuration files. These config templates will setup your stacks for you. See the `README.md` file in the scripts directory for more info: `cp configs/create-config.sample.yml configs/create-my-project-config.yml`
 3. Copy Parameters from production or staging stacks.
-	1. Select your primary AWS account credentials to copy stack parameters for the production Lara and Portal instances you are working on. `export AWS_PROFILE=concord`
-	2. Run the `create-stack` npm script to copy the stack paraemters your want. `npm run create-stack`
+	1. Select your primary AWS account credentials to copy stack parameters for the production LARA and Portal instances you are working on. `export AWS_PROFILE=concord`
+	2. Run the `create-stack` npm script to copy the stack parameters your want. `npm run create-stack`
 	3. Select the first menu item "Save stack params".  The script will show you a list of running stacks. Type-ahead search for the stack you want to copy into the QA environment.
 	4. Your parameters will be saved in the folder `stack-params`
 	5. Modify your `configs/create-my-project-config.yml file`
 		1. Update the `OriginalStack` parameter to point to the stack-params file you just downloaded.
-		2. Update any `ParameterModifications` you would like to ovveride. You will especially want to look at the domainName
+		2. Update any `ParameterModifications` you would like to override.
+      - Check or override these LARA params:
+        - HostName
+        - DomainNameBase
+        - CloudWatchLogGroup
+        - DbHost
+        - DatabaseSecurityGroupId
+        - Environment
+        - QAPortalSecret
+        - QAPortalURL
+      - Check or override these PORTAL params:
+        - ShortName
+        - DomainName
+        - CloudWatchLogGroup
+        - SiteURL
+        - Environment
+        - DbHost
+        - DatabaseSecurityGroupId
+        - AuthoringSiteURL
+        - S3SecreateAccessKey
+        - S3AccessKeyId
+        - ReportDomainName
+        - LogStashDbHost
+        - ClusterStackName
```

Comment:
The numbering in messed up in markdown after this section. I'm guessing it has to do with the indentation.

Reply:
I ran a Markdown Linter on this file and fixed several formatting issues. It turns out that the markdown for this section was valid, but the Github markdown engine seems to outdent unordered lists a bit when they are inside an ordered list. 

So I made the PARAMs list ordered even though they are non-sequential. I think it looks a bit better now.  Let me know what you think.